src/Avalonia.Themes.Fluent/Controls/HeaderedContentControl.xaml

```diff
@@ -0,0 +1,44 @@
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    x:ClassModifier="internal">
+  <ControlTheme x:Key="{x:Type HeaderedContentControl}"
+                TargetType="HeaderedContentControl">
+
+    <Setter Property="Padding" Value="3" />
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Border Name="PART_Border"
+                BorderBrush="{TemplateBinding BorderBrush}"
+                BorderThickness="{TemplateBinding BorderThickness}"
+                CornerRadius="{TemplateBinding CornerRadius}"
+                Background="{TemplateBinding Background}"
+                Padding="{TemplateBinding Padding}">
+          <Grid RowDefinitions="auto *">
+            <ContentPresenter Name="PART_HeaderPresenter"
```

Comment:
This is correctly a template part. It's used in code-behind here:

https://github.com/AvaloniaUI/Avalonia/blob/b2f2c49a19979946a5315dbce000f74baec0892a/src/Avalonia.Controls/Primitives/HeaderedContentControl.cs#L64C58-L68

However, that file is missing the template part attribute. Could you please modify it at the top to include:

`[TemplatePart("PART_HeaderPresenter", typeof(ContentPresenter))]`

See here for reference:

https://github.com/AvaloniaUI/Avalonia/blob/b2f2c49a19979946a5315dbce000f74baec0892a/src/Avalonia.Controls/ContentControl.cs#L16C1-L16C1

Note that the below "PART_ContentPresenter" is correct already and the template part attribute is inherited.

Reply:
Looks good, thanks. I can't mark these as resolved so if you want them to collapse go ahead and do so.